eventstore/src/event_store/generated/operations.rs

```diff
@@ -70,20 +84,25 @@ pub mod operations_client {
     impl<T> OperationsClient<T>
     where
         T: tonic::client::GrpcService<tonic::body::BoxBody>,
-        T::ResponseBody: Body + Send + 'static,
         T::Error: Into<StdError>,
+        T::ResponseBody: Body<Data = Bytes> + Send + 'static,
         <T::ResponseBody as Body>::Error: Into<StdError> + Send,
     {
         pub fn new(inner: T) -> Self {
             let inner = tonic::client::Grpc::new(inner);
             Self { inner }
         }
+        pub fn with_origin(inner: T, origin: Uri) -> Self {
+            let inner = tonic::client::Grpc::with_origin(inner, origin);
+            Self { inner }
+        }
```

Comment:
> The provided Uri will use only the scheme and authority parts as the path_and_query portion will be set for each method.

Is there a reason you would want to call `::new`, and not use the `uri`?

Reply:
Previous versions of the client used `new`, which is backed by `tonic::transport::Channel`. The reason why we are moving to `with_origin` is tonic remove advanced TLS configuration when using `Channel`. To maintain what we already provide, we have to build the http client from the ground up, hence why `with_origin`.